modules/tinymce/src/core/test/ts/browser/textpatterns/FindInlinePatternTest.ts

```diff
@@ -217,7 +224,7 @@ describe('browser.tinymce.textpatterns.FindInlinePatternTest', () => {
       it('TINY-8778: force only ** pattern and test return on not existing *** pattern', () => {
         const editor = hook.editor();
         setContentAndCursor(editor, '***x***', [ 0 ], 7);
-        const matches = getInlinePattern(editor, inlinePatternSet);
+        const matches = getInlinePattern(editor, inlinePatternSet as PatternSet);
```

Comment:
Why is the case needed here?

Reply:
Didn't clean up after the test was changed